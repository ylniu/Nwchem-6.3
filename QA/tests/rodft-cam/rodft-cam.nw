echo

start rodft-cam
title "N2+ 1.0977A B3LYP/6-31G** open-shell test"

charge 1

geometry
symmetry d2h
N 0.0 0.0 -0.54885
N 0.0 0.0 0.54885
end

basis
N library 6-31G**
end

dft
xc hfexch 1.00
cam 2000.0 cam_alpha 0.0 cam_beta 1.0 # HF limit
direct
mult 2
rodft
cgmin
end
task dft
