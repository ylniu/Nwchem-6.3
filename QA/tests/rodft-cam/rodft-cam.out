 argument  1 = rodft-cam.nw



============================== echo of input deck ==============================
echo

start rodft-cam
title "N2+ 1.0977A B3LYP/6-31G** open-shell test"

charge 1

geometry
symmetry d2h
N 0.0 0.0 -0.54885
N 0.0 0.0 0.54885
end

basis
N library 6-31G**
end

dft
xc hfexch 1.00
cam 2000.0 cam_alpha 0.0 cam_beta 1.0 # HF limit
direct
mult 2
rodft
cgmin
end
task dft
================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.3
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2012
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = orion
    program         = ../../../bin/LINUX64/nwchem
    date            = Fri May 24 00:26:05 2013

    compiled        = Thu_May_23_23:50:55_2013
    source          = /home/niri/nwchem/nwchem-6.3
    nwchem branch   = 6.3
    nwchem revision = 23861
    ga revision     = 10380
    input           = rodft-cam.nw
    prefix          = rodft-cam.
    data base       = ./rodft-cam.db
    status          = startup
    nproc           =        4
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107201 doubles =    100.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428802 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


                     N2+ 1.0977A B3LYP/6-31G** open-shell test
                     -----------------------------------------

 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)

 Turning off AUTOSYM since
 SYMMETRY directive was detected!


          ------
          auto-z
          ------


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 N                    7.0000     0.00000000     0.00000000    -0.54885000
    2 N                    7.0000     0.00000000     0.00000000     0.54885000

      Atomic Mass 
      ----------- 

      N                 14.003070


 Effective nuclear repulsion energy (a.u.)      23.6218321955

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000

      Symmetry information
      --------------------

 Group name             D2h       
 Group number             26
 Group order               8
 No. of unique centers     1

      Symmetry unique atoms

     1



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       1.09770


            XYZ format geometry
            -------------------
     2
 geometry
 N                     0.00000000     0.00000000    -0.54885000
 N                     0.00000000     0.00000000     0.54885000

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |  angstroms
 ------------------------------------------------------------------------------
    2 N                |   1 N                |     2.07435  |     1.09770
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          1
 ==============================================================================



  library name resolved from: environment
  library file name is: </home/niri/nwchem/nwchem-6.3/src/basis/libraries/>
  
                      Basis "ao basis" -> "" (cartesian)
                      -----
  N (Nitrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  4.17351100E+03  0.001835
  1 S  6.27457900E+02  0.013995
  1 S  1.42902100E+02  0.068587
  1 S  4.02343300E+01  0.232241
  1 S  1.28202100E+01  0.469070
  1 S  4.39043700E+00  0.360455

  2 S  1.16263580E+01 -0.114961
  2 S  2.71628000E+00 -0.169118
  2 S  7.72218000E-01  1.145852

  3 P  1.16263580E+01  0.067580
  3 P  2.71628000E+00  0.323907
  3 P  7.72218000E-01  0.740895

  4 S  2.12031300E-01  1.000000

  5 P  2.12031300E-01  1.000000

  6 D  8.00000000E-01  1.000000



 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 N                          6-31G**                  6       15   3s2p1d



                                 NWChem DFT Module
                                 -----------------


                     N2+ 1.0977A B3LYP/6-31G** open-shell test


  int_init: cando_txs set to always be  F
  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  restricted open shell.
          No. of atoms     :     2
          No. of electrons :    13
           Alpha electrons :     7
            Beta electrons :     6
          Charge           :     1
          Spin multiplicity:     2
          Use of symmetry is: on ; symmetry adaption is: on 
          Maximum number of iterations:  30
          This is a Direct SCF calculation.
          AO basis - number of functions:    30
                     number of shells:    12
          Convergence on energy requested: 1.00D-06
          Convergence on density requested: 1.00D-05
          Convergence on gradient requested: 5.00D-04

              XC Information
              --------------
                     Hartree-Fock (Exact) Exchange  1.000          

           Range-Separation Parameters        
           ---------------------------
           Alpha           :  0.00
           Beta            :  1.00
           Gamma           :******
           Short-Range HF  :     F

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL): 1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho: 1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul: 1.00D-08


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:        -108.58747649

 Renormalizing density from      14.00 to     13

      Non-variational initial energy
      ------------------------------

 Total energy =    -103.672717
 1-e energy   =    -180.042836
 2-e energy   =      52.748287
 HOMO         =      -0.962138
 LUMO         =      -0.507383


      Symmetry analysis of molecular orbitals - initial
      -------------------------------------------------


 !! scf_movecs_sym_adapt:   16 vectors were symmetry contaminated

  Symmetry fudging

 !! scf_movecs_sym_adapt:   16 vectors were symmetry contaminated

  Numbering of irreducible representations: 

     1 ag          2 au          3 b1g         4 b1u         5 b2g     
     6 b2u         7 b3g         8 b3u     

  Orbital symmetries:

     1 ag          2 b1u         3 ag          4 b1u         5 b3u     
     6 b2u         7 ag          8 b3g         9 b2g        10 b1u     
    11 ag         12 b3u        13 b2u        14 ag         15 b1u     
    16 b2g        17 b3g     



 ----------------------------------------------
         Quadratically convergent ROKS

 Convergence threshold     :          5.000E-04
 Maximum no. of iterations :           30
 Final Fock-matrix accuracy:          1.000E-07
 ----------------------------------------------


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1     -108.3448155680  5.34D-01  2.26D-01      0.2
                 2     -108.3558757765  7.64D-02  3.49D-02      0.3
                 3     -108.3562835882  1.40D-03  7.64D-04      0.6
                 4     -108.3562837612  4.81D-05  3.13D-05      0.7


         Total DFT energy =     -108.356283761249
      One electron energy =     -187.744303670875
           Coulomb energy =       55.766187714139
    Exchange-Corr. energy =        0.000000000000
 Nuclear repulsion energy =       23.621832195486

 Numeric. integr. density =        0.000000000000

     Total iterative time =      0.7s



                       DFT Final Molecular Orbital Analysis
                       ------------------------------------

 Vector    2  Occ=2.000000D+00  E=-1.619773D+01  Symmetry=b1u
              MO Center=  0.0D+00,  0.0D+00, -5.3D-17, r^2= 3.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.704064  1 N  s                 16     -0.704064  2 N  s          

 Vector    3  Occ=2.000000D+00  E=-1.908986D+00  Symmetry=ag
              MO Center= -2.5D-17, -6.7D-17,  2.6D-16, r^2= 4.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.364689  1 N  s                 17      0.364689  2 N  s          
     5      0.204084  1 N  pz                20     -0.204084  2 N  pz         
     6      0.173809  1 N  s                 21      0.173809  2 N  s          
     1     -0.161175  1 N  s                 16     -0.161175  2 N  s          

 Vector    4  Occ=2.000000D+00  E=-1.155147D+00  Symmetry=b1u
              MO Center= -9.4D-38, -1.3D-37, -2.3D-16, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.536348  1 N  s                 21     -0.536348  2 N  s          
     2      0.375797  1 N  s                 17     -0.375797  2 N  s          
     5     -0.214828  1 N  pz                20     -0.214828  2 N  pz         
     1     -0.150669  1 N  s                 16      0.150669  2 N  s          

 Vector    5  Occ=2.000000D+00  E=-1.039299D+00  Symmetry=b2u
              MO Center= -4.9D-33,  6.4D-17,  1.9D-17, r^2= 8.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.471696  1 N  py                19      0.471696  2 N  py         
     8      0.211316  1 N  py                23      0.211316  2 N  py         

 Vector    6  Occ=2.000000D+00  E=-1.039299D+00  Symmetry=b3u
              MO Center=  2.6D-17,  1.1D-47,  1.4D-17, r^2= 8.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.471696  1 N  px                18      0.471696  2 N  px         
     7      0.211316  1 N  px                22      0.211316  2 N  px         

 Vector    7  Occ=1.000000D+00  E=-8.312300D-01  Symmetry=ag
              MO Center= -6.2D-35,  4.3D-19,  1.2D-16, r^2= 1.1D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.499031  1 N  pz                20     -0.499031  2 N  pz         
     6     -0.262742  1 N  s                 21     -0.262742  2 N  s          
     9      0.156291  1 N  pz                24     -0.156291  2 N  pz         

 Vector    8  Occ=0.000000D+00  E=-2.359439D-01  Symmetry=b3g
              MO Center= -1.3D-52,  8.7D-38,  6.9D-17, r^2= 1.3D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      0.571375  1 N  py                23     -0.571375  2 N  py         
     4      0.496219  1 N  py                19     -0.496219  2 N  py         

 Vector    9  Occ=0.000000D+00  E=-2.359439D-01  Symmetry=b2g
              MO Center= -2.6D-18, -1.1D-33,  1.4D-17, r^2= 1.3D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      0.571375  1 N  px                22     -0.571375  2 N  px         
     3      0.496219  1 N  px                18     -0.496219  2 N  px         

 Vector   10  Occ=0.000000D+00  E= 2.505689D-01  Symmetry=b1u
              MO Center=  5.3D-20, -5.0D-32,  1.1D-15, r^2= 3.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      3.864570  1 N  s                 21     -3.864570  2 N  s          
     9      2.601424  1 N  pz                24      2.601424  2 N  pz         
     2      0.238216  1 N  s                 17     -0.238216  2 N  s          

 Vector   11  Occ=0.000000D+00  E= 4.308479D-01  Symmetry=ag
              MO Center=  7.4D-34, -1.4D-19, -5.6D-17, r^2= 1.8D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     9      0.788308  1 N  pz                24     -0.788308  2 N  pz         
     2     -0.594033  1 N  s                 17     -0.594033  2 N  s          
     6      0.526252  1 N  s                 21      0.526252  2 N  s          
     5     -0.267856  1 N  pz                20      0.267856  2 N  pz         
    15     -0.193492  1 N  dzz               30     -0.193492  2 N  dzz        

 Vector   12  Occ=0.000000D+00  E= 4.706138D-01  Symmetry=b3u
              MO Center= -6.4D-19, -4.9D-36,  1.0D-17, r^2= 1.9D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7     -0.633748  1 N  px                22     -0.633748  2 N  px         
     3      0.607855  1 N  px                18      0.607855  2 N  px         
    12      0.165458  1 N  dxz               27     -0.165458  2 N  dxz        

 Vector   13  Occ=0.000000D+00  E= 4.706138D-01  Symmetry=b2u
              MO Center=  2.0D-48,  2.4D-18, -1.0D-17, r^2= 1.9D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8     -0.633748  1 N  py                23     -0.633748  2 N  py         
     4      0.607855  1 N  py                19      0.607855  2 N  py         
    14      0.165458  1 N  dyz               29     -0.165458  2 N  dyz        

 Vector   14  Occ=0.000000D+00  E= 4.812514D-01  Symmetry=ag
              MO Center= -1.3D-20,  9.0D-20, -1.2D-16, r^2= 2.7D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.973271  1 N  s                 21      0.973271  2 N  s          
     2     -0.599711  1 N  s                 17     -0.599711  2 N  s          
     9     -0.578254  1 N  pz                24      0.578254  2 N  pz         
     5      0.415107  1 N  pz                20     -0.415107  2 N  pz         

 Vector   15  Occ=0.000000D+00  E= 6.388523D-01  Symmetry=b3g
              MO Center=  2.7D-35, -1.6D-19,  9.5D-18, r^2= 2.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      1.221487  1 N  py                23     -1.221487  2 N  py         
     4     -0.688530  1 N  py                19      0.688530  2 N  py         

 Vector   16  Occ=0.000000D+00  E= 6.388523D-01  Symmetry=b2g
              MO Center=  4.3D-17,  2.4D-34, -8.3D-17, r^2= 2.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      1.221487  1 N  px                22     -1.221487  2 N  px         
     3     -0.688530  1 N  px                18      0.688530  2 N  px         

 Vector   17  Occ=0.000000D+00  E= 6.702852D-01  Symmetry=b1u
              MO Center= -9.7D-17,  1.1D-18,  2.8D-16, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      1.690290  1 N  s                 21     -1.690290  2 N  s          
     5      0.678134  1 N  pz                20      0.678134  2 N  pz         
    15     -0.353878  1 N  dzz               30      0.353878  2 N  dzz        
     2     -0.308527  1 N  s                 17      0.308527  2 N  s          
     9      0.191912  1 N  pz                24      0.191912  2 N  pz         


 Task  times  cpu:        0.8s     wall:        1.1s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  401      401     3004     1184     1478        0        0      799     
number of processes/call 1.51e+00 1.22e+00 1.20e+00 0.00e+00 0.00e+00
bytes total:             3.06e+06 1.06e+06 1.53e+06 0.00e+00 0.00e+00 6.39e+03
bytes remote:            1.37e+06 3.18e+05 7.30e+05 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 82008 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        14	        17
	current total bytes		         0	         0
	maximum total bytes		     80096	  22509912
	maximum total K-bytes		        81	     22510
	maximum total M-bytes		         1	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata,
     F. Aquino, S. Hirata, M. T. Hackler, J. Mullin, P. Nichols, R. Peverati,
    J. Pittner, Y. Zhao, P.-D. Fan, R. J. Harrison, M. Dupuis, D. Silverstein,
    D. M. A. Smith, J. Nieplocha, V. Tipparaju, M. Krishnan, B. E. Van Kuiken,
        A. Vazquez-Mayagoitia, L. Jensen, M. Swart, Q. Wu, T. Van Voorhis,
     A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros, G. I. Fann,
     H. Fruchtl, J. Garza, K. Hirao, R. Kendall, J. A. Nichols, K. Tsemekhman,
      K. Wolinski, J. Anchell, D. Bernholdt, P. Borowski, T. Clark, D. Clerc,
      H. Dachsel, M. Deegan, K. Dyall, D. Elwood, E. Glendening, M. Gutowski,
      A. Hess, J. Jaffe, B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin,
   R. Littlefield, X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing,
   K. Glaesemann, G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe,
                                A. Wong, Z. Zhang.

 Total times  cpu:        0.8s     wall:        1.3s
